### DISCUSSION TITLE
Support for function to get raw pcm data

### DISCUSSION THREAD

**Discussion by yeonns** [euphony-io/euphony#129] [Ideas]
Now, we can get string result when we ```listen()``` with EuRxManager.
But, I want to get raw pcm deta result.
With these raw pcm data, we can utilize them as AI dataset. :smile: 

So, how about supporting this feature?

**Comment by designe** [comment]
It is an interesting idea 👍 
Let me implement this export feature & reflect it in the next patch :)

**Reply by yeonns** [reply]
Thank you! :smile:
